app/src/androidTest/java/com/example/android/architecture/blueprints/todoapp/TasksActivityTest.kt

```diff
@@ -64,8 +67,16 @@ class TasksActivityTest {
 
     @Before
     fun init() {
-        repository = ServiceLocator.provideTasksRepository(getApplicationContext())
-        repository.deleteAllTasksBlocking()
+        repository =
+            ServiceLocator.provideTasksRepository(
+                getApplicationContext()
+            )
+        // TODO why is this run blocking instead of runBlockingTest? runBlockingTest causes
```

Comment:
Because it's there to be used by anyone -- test or not

Reply:
This isn't a test, so shouldn't use runBlockingTest.